lms/views/helpers/_via.py

```diff
@@ -18,9 +18,8 @@ def via_url(request, document_url, content_type=None):
     :param content_type: Either "pdf" or "html" if known, None if not
     :return: A URL string
     """
-
-    doc = ViaDoc(document_url, content_type)
-
     return ViaClient(
-        service_url=request.registry.settings["via_url"], host_url=request.host_url
-    ).url_for(doc)
+        request.registry.settings["via_url"],
+        request.host_url,
+        request.registry.settings["via_secret"],
```

Comment:
I think as we add more options here it's good to keep the kwargs to say what they all are. Easy to swap two and not be able to spot it.

I'll be adding another one for ViaHTML pretty soon.

This probably should be a service... but who's got time for that?